fundingmanager.go

```diff
@@ -1189,6 +1240,13 @@ func (f *fundingManager) handleFundingCreated(fmsg *fundingCreatedMsg) {
 		return
 	}
 
+	// At this point we have received the last funding message from the
+	// initiating peer before the funding transaction will be broadcast so
+	// set the timestamp to the zero value. This indicates to the zombie
+	// sweeper that this reservation should not be pruned, as it is now
+	// waiting for the funding transaction
+	resCtx.lastUpdated = time.Time{}
```

Comment:
do this first after the last message is sent to the peer.

Reply:
Also consider placing these timestamp changes into methods like
`updateLastUpdated(resCtx)` and `lockLastUpdated(resCtx)` or similar.